### PR TITLE
Feat: Add suport to host config

### PR DESCRIPTION
**Pull request by jpwerka**
Feat: Add suport to host config
https://github.com/simonh1000/angular-http-server/issues/60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +161,7 @@ The `--https` or `--ssl` flags are intended for **development and/or testing pur
 
 ## Changelog
 
+-   1.12.0 - adds host support (thanks jpwerka)
```
Comment (simonh1000) [inline README.md:164]: 1.20

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "angular-http-server",
-    "version": "1.11.3",
+    "version": "1.12.0",
```
Comment (simonh1000) [inline package.json:3]: 1.20

**Comment by simonh1000** [comment]
thanks

**Comment by jpwerka** [comment]
@simonh1000 
Is there any prediction when the updated version of the library will be published on npm?

**Comment by simonh1000** [comment]
done

**Pull request merged**